src/pages/_app.tsx

```diff
@@ -85,26 +91,65 @@ function MyApp({ Component, pageProps }) {
           sizes="192x192"
           href="/assets/icon/android-icon-192x192.png"
         />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="32x32"
-          href="/assets/icon/favicon-32x32.png"
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="96x96"
-          href="/assets/icon/favicon-96x96.png"
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="16x16"
-          href="/assets/icon/favicon-16x16.png"
-        />
+        {router.route.startsWith('/gen2') ? (
```

Comment:
Not a blocker for me, but if you are in this file making updates and unless anyone objects I think it would me more readable to combine all of these conditionals into just one.

Reply:
Agreed and fixed.